modules/ocf/manifests/packages/firefox.pp

```diff
@@ -1,27 +1,13 @@
 class ocf::packages::firefox {
   $browser_homepage = lookup('browser_homepage')
 
-  class { 'ocf::packages::firefox::apt':
-    stage => first,
-  }
-
-  # TODO: switch to mozilla.debian.net once they support stretch.
```

Comment:
Do we still want to switch to this sometime? I suppose we'd rather use the upstream version

Reply:
Well, they're not going to support it until at least buster.

Reply:
I guess we can add A TODO for when buster comes out...

Reply:
Hmm, alright